tools/pfif-tools/app/pfif_validator.py

```diff
@@ -762,7 +762,7 @@ def validate_linked_records_matched(self):
           link_field = linking_note.find(
               self.tree.add_namespace_to_tag('linked_person_record_id'))
           messages.append(self.make_message(
-              'There is an asymmetric linked record.  That is, a note has a'
+              'There is an asymmetric linked record.  That is, a note has a '
```

Comment:
Only 1 space after a period.
```suggestion
              'There is an asymmetric linked record. That is, a note has a '
```

Reply:
Done.